src/pages/reference/starters.md

```diff
@@ -18,20 +18,42 @@ This repo is open source and contributions (new starters, updating existing ones
 
 Starters can point at any public repository.
 
-## Railway Button
+## Updating Starters
+
+Every time you visit your project on Railway, we will check to see if the project it is based on has been updated by its maker.
+
+If it has, we will prompt you to update your project. On confirmation, we will create a branch on Github and open a PR deployment for you on Railway. This way, we don’t replace your production deployment and you can test things out within the PR deploy.
+
+Once you’re happy with the changes, you can merge the PR and your production deployment will be updated to the latest version.
+
+## Templates
 
-The button allows you to offer a 1-Click deploy on Railway button.
+Templates are user created starters that can be deployed at the clock of a button.
+
+The button allows you to offer a 1-Click deploy on Railway experience.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1656470421/docs/template-editor_khw8n6.png"
+alt="Screenshot of Template Editor"
+layout="intrinsic"
+width={609} height={520} quality={80} />
 
 ![Railway Button](https://railway.app/button.svg)
 
 Configure your own button at
 [railway.app/button](https://railway.app/button) where you can define the repo
 to deploy, plugins to install, and required env vars.
 
-## Updating Starters
+### Managing Templates
 
-Every time you visit your project on Railway, we will check to see if the project it is based on has been updated by its maker.
+<Image src="https://res.cloudinary.com/railway/image/upload/v1656470419/docs/template-manager_ki6byi.png"
+alt="Screenshot of Expanded Project Usage Pane"
+layout="intrinsic"
+width={973} height={562} quality={80} />
 
-If it has, we will prompt you to update your project. On confirmation, we will create a branch on Github and open a PR deployment for you on Railway. This way, we don’t replace your production deployment and you can test things out within the PR deploy.
+You can see all of your templates that you created within your [Account's Templates page](https://railway.app/account/templates). You can edit your Template at any time.
 
-Once you’re happy with the changes, you can merge the PR and your production deployment will be updated to the latest version.
+## Deploy a Certain Commit
```

Comment:
You can't deploy a specific commit. But you can deploy the last known stable version.

```suggestion
## Deploy a Stable Version
```